baseplate/clients/requests.py

```diff
@@ -210,7 +210,9 @@ def send(self, request: PreparedRequest, **kwargs: Any) -> Response:
             "http.url": request.url,
             "http.slug": self.client_name if self.client_name is not None else self.name,
         }
-        with self.span.make_child(f"{self.name}.request").with_tags(tags) as span:
+        with self.span.make_child(f"{self.name}.request", component_name="http_client").with_tags(
```

Comment:
The protocol tag forces the metrics to come with this name, but I thought I'd be consistent